code/modules/mining/machine_stacking.dm

```diff
@@ -35,6 +33,13 @@
 
 	user << browse(dat, "window=console_stacking_machine")
 
+/obj/machinery/mineral/stacking_unit_console/multitool_act(mob/living/user, obj/item/I)
+	var/obj/item/multitool/M = I

```

Comment:
You usually want to declare vars like this *after* doing a typecheck.

Reply:
I tried, but it broke when I did it that way. as in, it stopped working entirely

Reply:
Don't know why, it's working fine in the other method.

Reply:
fucking byond, it works fine now, but last time it was throwing compiler errors at me.
next commit should fix all issues